tests/test_mail.py

```diff
@@ -391,9 +391,31 @@ def test_mail_recipients_to(requests_mock):
         ],
     }
     m = setup_mail_mock(requests_mock, conf)
+
     m.to = ["@novem_demo", "@test"]
     assert m.to == "@novem_demo\n@test"
+
     m.to = "@novem_demo\n@test"
+    assert m.to == "@novem_demo\n@test"
+
+    conf = {
+        "mail_id": "test_mail",
+        "reqs": [
+            [
+                "get",
+                "/recipients/to",
+                "",
+            ],
+            ["post", "/recipients/to", None],
+        ],
+    }
+    m = setup_mail_mock(requests_mock, conf)
```

Comment:
I think this function does too much - it's not very obvious what this test actually does now. We can revisit.

Reply:
Maybe split these additions into new test cases.

Reply:
Basically it mocks the expected server response on `GET` and expected request server query on `POST`. But yes, not super easy to understand.

Not 100% sure how to make it much easier though, as it does exercise meaningful code paths